specification/dataprotection/resource-manager/Microsoft.DataProtection/stable/2023-01-01/examples/PolicyCRUD/CreateOrUpdateBackupPolicy.json

```diff
@@ -0,0 +1,200 @@
+    "resourceGroupName": "000pikumar",
+    "vaultName": "PrivatePreviewVault",
+    "backupPolicyName": "OSSDBPolicy",
+    "api-version": "2023-01-01",
+    "parameters": {
+      "properties": {
+        "policyRules": [
+          {
+            "backupParameters": {
+              "backupType": "Full",
+              "objectType": "AzureBackupParams"
+            },
+            "trigger": {
+              "schedule": {
+                "repeatingTimeIntervals": [
+                  "R/2019-11-20T08:00:00-08:00/P1W"
+                ]
+              },
+              "taggingCriteria": [
+                {
+                  "tagInfo": {
+                    "tagName": "Default"
+                  },
+                  "taggingPriority": 99,
+                  "isDefault": true
+                },
+                {
+                  "tagInfo": {
+                    "tagName": "Weekly"
+                  },
+                  "taggingPriority": 20,
+                  "isDefault": false,
+                  "criteria": [
+                    {
+                      "scheduleTimes": [
+                        "2019-03-01T13:00:00Z"
+                      ],
+                      "daysOfTheWeek": [
+                        "Sunday"
+                      ],
+                      "objectType": "ScheduleBasedBackupCriteria"
+                    }
+                  ]
+                }
+              ],
+              "objectType": "ScheduleBasedTriggerContext"
+            },
+            "dataStore": {
+              "dataStoreType": "VaultStore",
+              "objectType": "DataStoreInfoBase"
+            },
+            "name": "BackupWeekly",
+            "objectType": "AzureBackupRule"
+          },
+          {
+            "lifecycles": [
+              {
+                "sourceDataStore": {
+                  "dataStoreType": "VaultStore",
+                  "objectType": "DataStoreInfoBase"
+                },
+                "deleteAfter": {
+                  "objectType": "AbsoluteDeleteOption",
+                  "duration": "P1W"
+                }
+              }
+            ],
+            "isDefault": true,
+            "name": "Default",
+            "objectType": "AzureRetentionRule"
+          },
+          {
+            "lifecycles": [
+              {
+                "sourceDataStore": {
+                  "dataStoreType": "VaultStore",
+                  "objectType": "DataStoreInfoBase"
+                },
+                "deleteAfter": {
+                  "objectType": "AbsoluteDeleteOption",
+                  "duration": "P12W"
+                }
+              }
+            ],
+            "isDefault": false,
+            "name": "Weekly",
+            "objectType": "AzureRetentionRule"
+          }
+        ],
+        "datasourceTypes": [
+          "OssDB"
+        ],
+        "objectType": "BackupPolicy"
+      }
+    }
+  },
+  "responses": {
+    "200": {
```

Comment:
Should also include a sample 201 response, since this is a create or update sample

Reply:
This operation is always performed in sync time

we do not support async operation for policy create/update.